subcommands/status/cmd.go

```diff
@@ -61,10 +61,10 @@ func showStatus(cmd *cobra.Command, args []string) {
 		}
 		fmt.Println("\n## Tag:", tag.Name)
 		// Tabby doesn't indent (or at least easily) so:
-		fmt.Println("\tTARGET  DEVICES  DETAILS")
-		fmt.Println("\t------  -------  -------")
+		fmt.Println("\tTARGET  DEVICES  (RE-)INSTALLING DETAILS")
+		fmt.Println("\t------  -------  --------------- -------")
 		for _, tgt := range tag.Targets {
-			fmt.Printf("\t%-6d  %-7d  `fioctl targets show %d`\n", tgt.Version, tgt.Devices, tgt.Version)
+			fmt.Printf("\t%-6d  %-7d  %-15d `fioctl targets show %d`\n", tgt.Version, tgt.Devices, tgt.Reinstalling, tgt.Version)
```

Comment:
Does `Reinstalling` print true or false or a list of apps are being reinstalled?